rosetta-cli-config/docker/docker-entrypoint.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+# Use this script to enable host.docker.internal on Docker for linux.
+# See https://github.com/bufferings/docker-access-host
+
+HOST_DOMAIN="host.docker.internal"
+ping -q -c1 $HOST_DOMAIN >/dev/null 2>&1
+if [ $? -ne 0 ]; then
+  HOST_IP=$(ip route | awk 'NR==1 {print $3}')
+  echo -e "$HOST_IP\t$HOST_DOMAIN" >>/etc/hosts
+fi
+
+cat /etc/hosts
+echo "patched host.docker.internal"
+
+# wait for things to settle down
+sleep 20
```

Comment:
Is this needed? This adds a minimum of 20 extra seconds to run this test suite right? I'm not sure why this should be necessary.

Reply:
This is not necessary, It must have slipped in while debugging. I have updated it.